superset-frontend/src/views/CRUD/data/database/DatabaseModal/DatabaseConnectionForm/TableCatalog.tsx

```diff
@@ -34,7 +34,6 @@ export const TableCatalog = ({
 }: FieldPropTypes) => {
   const tableCatalog = db?.catalog || [];
   const catalogError = validationErrors || {};
-
```

Comment:
nit: Could we add this space back since it is the only change in this file.

Reply:
yep